dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.9.12-buster as base
```

Comment:
Minor, but this file is typically called `Dockerfile` with a capital D - it often won't affect you, but some systems or tools might rely on that standard, so it's generally worth following!